Source/ZoomNet/Models/IssuesOfZoomRooms.cs

```diff
@@ -34,7 +34,7 @@ public class IssuesOfZoomRooms
 		/// </summary>
 		/// <value>The Issue name.</value>
 		[JsonProperty(PropertyName = "issue_name")]
-		public IssueType IssueType { get; set; }
+		public IssueType IssueName { get; set; }
```

Comment:
I realize that the json element from Zoom is called "issue_name", but I think "IssueType" or simply "Type" is a more suitable name for this property in our C# model to describe what this property contains.